app/templates/gulpfile.js

```diff
@@ -3,7 +3,7 @@
 'use strict';
 
 var gulp = require('gulp'),
-    gutil = require('gulp-util'),
+//    gutil = require('gulp-util'),
```

Comment:
Why is this commented? If it is not ysed we can remove no need to keep it commented